projects/mtg/include/MTGRules.h

```diff
@@ -58,6 +58,10 @@ class MTGEventBonus: public PermanentAbility
     int Angel[2];
     bool dragonbonusgranted[2];
     int dragon[2];
+	bool eldrazibonusgranted[2];
+	int eldrazi[2];
+	bool werewolfbonusgranted[2];
+	int werewolf[2];
```

Comment:
yeah, i know, i dont have my original set up which had the tabs set up correctly, and I dont remember what the correct settings are for wagic project. the thread on the forums was not very specific as to where you set this up.